custom_dc/README.md

```diff
@@ -6,6 +6,17 @@ custom to a third party's need.
 This document illustrates development and deployment details for a custom Data
 Commons instance.
 
+## Custom and Base Data Commons
+
+<img src="https://github.com/datacommonsorg/website/assets/1221814/63efdb88-580c-4bc2-967b-5dce61844eea" 
```

Comment:
nit: there appears to be a lot of whitespace around the image, any chance of trimming?

Reply:
It seems to add a paragraph tag around it. Tried a few things but could not get rid of it.